k9mail/src/main/java/com/fsck/k9/notification/NotificationHolder.java

```diff
@@ -0,0 +1,13 @@
+package com.fsck.k9.notification;
+
+
+class NotificationHolder {
```

Comment:
the class names `NotificationHolder` and `NotificationsHolder` are fairly difficult to quickly tell apart, particularly since they are often used close to each other